packages/communicator-email-ses/README.md

```diff
@@ -0,0 +1,79 @@
+# DBOS AWS Simple Email Service (SES) Communicator
+
+This is a [DBOS](https://docs.dbos.dev/) [communicator](https://docs.dbos.dev/tutorials/communicator-tutorial) for sending email using the [Amazon Web Services Simple Email Service](https://aws.amazon.com/ses/).
+
+## Getting Started
+In order to send emails with SES, it is necessary to:
+- Register with AWS and create access keys for SES. (See [Setting up SES](https://docs.aws.amazon.com/ses/latest/dg/setting-up.html) in AWS documentation.)
+- Verify a sending domain and destination addresses. (SES will initially be in "sandbox mode", which constrains email sending to be from a validated domain, to a validated email address.  See [Verified identities](https://docs.aws.amazon.com/ses/latest/dg/setting-up.html) in AWS documentation.)
+
+## Configuring a DBOS Application with AWS SES
+First, ensure that the DBOS SES communicator is installed into the application:
+```
+npm install --save @dbos-inc/communicator-email-ses
+```
+
+Second, ensure that the communicator is imoprted and exported from an application entrypoint source file:
+```typescript
+import { SendEmailCommunicator } from "@dbos-inc/communicator-email-ses";
+export { SendEmailCommunicator };
+```
+
+Third, place appropriate configuration into the [`dbos-config.yaml`](https://docs.dbos.dev/api-reference/configuration) file; the following example will pull the AWS information from the environment:
+```yaml
+application:
+  aws_ses_configurations: aws_config
+  aws_config:
+    aws_region: ${AWS_REGION}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
```

Comment:
Also add `ses_to_address` and `ses_from_address` to this config?

Reply:
No.  The to and from addresses are provided by the application.  Getting them from the environment by way of the config file is something the unit test does, but is not what an app is expected to do.